Update: Add example of multiple connections between models (#4017)
aws-amplify/docs: src/pages/cli/graphql/data-modeling.mdx

}
```

### Create multiple relationships between two models

> You need to explicitly specifying the connection field names, if relational directives are used to create two connections of the same type between the two models.

Reviewer: ```suggestion
You need to explicitly specify the connection field names if relational directives are used to create two connections of the same type between the two models.
```

> If relational directives are used to create two of the same type of connection between the same two models, the transformer is not able to disambiguate the connections. However, by explicitly specifying the connection field names this ambiguity can be mitigated.

Reviewer 2: ```suggestion
You need to explicitly specifying the connection field names, if relational directives are used to create two connections of the same type between the two models.
```

> If relational directives are used to create two of the same type of connection between the same two models, the transformer is not able to disambiguate the connections. However, with some additional specification it is doable. This avoids ambiguity by naming the connection fields directly instead of allowing the auto generated fields to have the same name and collide.

Reviewer: ```suggestion
If relational directives are used to create two of the same type of connection between the same two models, the transformer is not able to disambiguate the connections. However, by explicitly specifying the connection field names this ambiguity can be mitigated.
```

```graphql
type Individual @model {
  id: ID!
  homeAddress: Address @hasOne
  shippingAddress: Address @hasOne
}

type Address @model {
  id: ID!
  homeIndividualID: ID @index(name: "homeIndex")

>   shippingIndividualID: ID @index(name: "shipIndex")

Reviewer: I think the use of `@index` here distracts from what the example is about. What do you think?

Author: Yeah that's fair, I can cut it out of the schema

>   #homeIndividual: Individual @belongsTo(fields: ["homeIndividualID"])

Reviewer: Are these supposed to be commented out?
